README.md

```diff
@@ -1,135 +1,131 @@
+  cpu?: number;
+  memoryMiB?: number;
+  minInstances?: number;
+  maxInstances?: number;
+}
 
-In your `firebase.json` you can alter the configuration of the code-generated Cloud Function by editing the "frameworksBackend"
-option. "frameworksBackend" takes the same options as [firebase-functions/v2/https.httpsOptions](https://firebase.google.com/docs/reference/functions/2nd-gen/node/firebase-functions.https.httpsoptions)
-though JSON-serializable. E.g,
+interface EnvVarConfig {
+  variable: string;
+  value: string;
+  availability: 'RUNTIME'
+}
 
+```
 
-```json
-{
-  "hosting": {
-    "source": ".",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],
-    "frameworksBackend": {
-      "region": "us-central1",
-      "minInstances": 1,
-      "maxInstances": 10
-    }
-  }
+| Field  | Type | Description |
+| ---------- | ------- | - |
+| runCommand | string |Command to start the server (e.g. "node dist/index.js"). Assume this command is run from the root dir of the workspace.This should be the productionized version of the server start command. |
+| environmentVariables| EnvVarConfig[] | Environment variables present in the server execution environment.|
+| concurrency | number | The maximum number of concurrent requests that each server instance can receive.|
+| cpu | number |The number of CPUs used in a single server instance.. |
+| memoryMiB | number | The amount of memory available for a server instance.|
+| minInstance | number |The limit on the minimum number of function instances that may coexist at a given time. |
+| MaxInstance | number | The limit on the maximum number of function instances that may coexist at a given time.|
+| EnvVarConfig.variable | string |Name of the environment variable |
+| EnvVarConfig.value | string |Value associated with the environment variable |
+| EnvVarConfig.availability | RUNTIME | Where the variable will be available, for now this will always be RUNTIME |
+
+Many of these fields are shared with apphosting.yaml as well so see [our documentation](https://firebase.google.com/docs/reference/apphosting/rest/v1beta/projects.locations.backends.builds#runconfig) for additional context.
+
+### Metadata
+
+```typescript
+interface Metadata {
+  adapterPackageName: string;
+  adapterVersion: string;
+  framework: string;
+  frameworkVersion?: string;
 }
+
 ```
 
-# Contributors
+| Field  | Type | Description |
+| ---------- | ------- | - |
+| adapterPackageName | string |Name of the adapter (this should be the npm package name) |
+| adapterVersion| string | Version of the adapter|
+| framework | string | Name of the framework that is being supported|
+| frameworkVersion | string |Version of the framework that is being supported |
 
-We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to
-follow. [See CONTRIBUTING](./CONTRIBUTING.md).
+Putting this all together a sample bundle.yaml would look as follows:
```

Comment:
Suggest

"Putting this all together in a sample bundle.yaml would look like the following"

or maybe just

"Here is a sample bundle.yaml file putting all this together:"

or really:

"Here is a sample bundle.yaml file demonstrating [what we are trying to demonstrate].

I'll stop now :)